README.md

```diff
@@ -336,11 +344,13 @@ Given a walker, TypeScript's parser visits the AST using the visitor pattern. So
 We still need to hook up this new rule to TSLint. First make sure to compile `noImportsRule.ts`:
 
 ```bash
-tsc -m commonjs --noImplicitAny noImportsRule.ts node_modules/tslint/lib/tslint.d.ts
+tsc --noImplicitAny noImportsRule.ts
 ```
 
 Then, if using the CLI, provide the directory that contains this rule as an option to `--rules-dir`. If using TSLint as a library or via `grunt-tslint`, the `options` hash must contain `"rulesDirectory": "..."`. If you run the linter, you'll see that we have now successfully banned all import statements via TSLint!
 
+Finally, add a line to your [`tslint.json` config file][0] for each of your custom rules.
```

Comment:
suggestion: something along the lines of "... enable each custom rule in your tslint.json config file"